Assets/MixedRealityToolkit.Providers/WindowsVoiceInput/WindowsDictationInputProvider.cs

```diff
@@ -31,11 +29,10 @@ public class WindowsDictationInputProvider : BaseDeviceManager, IMixedRealityDic
             uint priority = DefaultPriority,
             BaseMixedRealityProfile profile = null) : base(registrar, name, priority, profile) { }
 
-        /// <summary>
-        /// Is the Dictation Manager currently running?
-        /// </summary>
+        /// <inheritdoc />
         public bool IsListening { get; private set; } = false;
 
+#if UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_EDITOR_WIN
```

Comment:
won't this be a compile error if those defines are not set because the braces are not matching anymore?

Reply:
Oof yeah, good catch. It causes a few other problems too...working through them now.

Reply:
I moved them around. The "other problems" I mentioned turned out to be unimportant.